src/engine/models/bounding_box.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+# ______       ____   ___
+#   |     \/   ____| |___|    
+#   |     |   |   \  |   |       
+#-----------------------------------------------------------------------
+# Copyright 2020, tyra - https://github.com/h4570/tyra
+# Licenced under Apache License 2.0
+# Michał Mostowik <mostek3pl@gmail.com>
+*/
+
+#include "../include/models/bounding_box.hpp"
+#include "../include/utils/debug.hpp"
+
+BoundingBox::BoundingBox(Vector3 *t_vertices)

```

Comment:
I think that will be good idea to inform the user, that this array **must** be the size of 8.
What I mean is this style of comment:
https://github.com/h4570/tyra/blob/bb0ec257b2cd4c9aa317043b47a5f998a92dce24/src/engine/include/modules/audio.hpp#L47